src/dotnet/commands/dotnet-tool/update/LocalizableStrings.resx

```diff
@@ -186,4 +183,16 @@
   <data name="UpdateToolFailed" xml:space="preserve">
     <value>Tool '{0}' failed to update due to the following:</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="VersionOptionName" xml:space="preserve">
+    <value>VERSION</value>
+  </data>
+  <data name="VersionOptionDescription" xml:space="preserve">
+    <value>The specific version of the tool package to update to.</value>
```

Comment:
```suggestion
    <value>The specific version of the tool package to update to. The version must be newer than the currently installed version.</value>
```

Reply:
So we're parsing the option using `VersionRange` below, which treats version `1.2.3.4` as a minimum (inclusive) for the update and not a specific version.

If we want this to be specific, then we should parse it with `NugetVersion.TryParse` and then construct a `VersionRange` with min and max set to the version, inclusive.

Reply:
to "The version range of the tool package to update to"